metricbeat/module/php_fpm/_meta/config.yml

```diff
@@ -0,0 +1,6 @@
+- module: php_fpm
```

Comment:
We currently have all module configs commented out which are not enabled by default. This might change in the near future, but for consistency it should be commented out here